extensions/amp-story/1.0/amp-story.js

```diff
@@ -1632,6 +1635,9 @@ export class AmpStory extends AMP.BaseElement {
         actions.execute(this.sidebar_, 'open', /* args */ null,
             /* source */ null, /* caller */ null, /* event */ null,
             ActionTrust.HIGH);
+      } else if (this.sidebar_ && !sidebarState &&
+                    !this.maskElement_.hasAttribute('hidden')) {
+        this.closeOpacityMask_();
```

Comment:
This code is not needed either, and breaks the MutationObserver disconnect behavior. Please remove.